src/api/config/environments/development.rb

```diff
@@ -8,8 +8,8 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  # Do not eager load code on boot.
-  config.eager_load = false
+  # Eager load code on boot.
+  config.eager_load = true
```

Comment:
@Ana06 see my last post about pluralising the project controller. Though I think thats my favorite solution too.. @ChrisBr what do you think? Do you still prefer to use `::Project`?